examples/operator-rbac/2-client.ipynb

```diff
@@ -0,0 +1,918 @@
+    "- See the example will use service account from pod [Example](https://github.com/feast-dev/feast/blob/master/examples/rbac-remote/client/k8s/)\n",
+    "\n",
+    "### Outside a Kubernetes Pod (Local Machine)\n",
+    "If running Feast outside of Kubernetes, authentication requires setting the token manually:\n",
+    "```sh\n",
+    "export LOCAL_K8S_TOKEN=\"your-service-account-token\"\n",
+    "```\n",
+    "Feast will use this token for authentication.\n",
+    "\n",
+    "Reference:  \n",
+    "[Feast Authentication via `LOCAL_K8S_TOKEN`](https://github.com/feast-dev/feast/blob/master/sdk/python/feast/permissions/client/kubernetes_auth_client_manager.py#L50)"
+   ],
+   "id": "bb0145c9c1f6ebcc"
+  },
+  {
+   "metadata": {},
+   "cell_type": "markdown",
+   "source": [
+    "## Test Cases\n",
+    "| User Type       | ServiceAccount               | RoleBinding Assigned | Expected Behavior in output                                |\n",
+    "|----------------|-----------------------------|----------------------|------------------------------------------------------------|\n",
+    "| **Read-Only**  | `feast-user-sa`              | `feast-reader`       | Can **read** from the feature store, but **cannot write**. |\n",
+    "| **Unauthorized** | `feast-unauthorized-user-sa` | _None_               | **Access should be denied** in `test.py`.                  |\n",
+    "| **Admin**      | `feast-admin-sa`             | `feast-writer`       | Can **read and write** feature store data.                 |"
+   ],
+   "id": "160681ba4ab3c2c5"
+  },
+  {
+   "metadata": {},
+   "cell_type": "markdown",
+   "source": "###  Feature Store settings",
+   "id": "6590c081efb1fe3c"
+  },
+  {
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2025-03-05T18:51:44.898752Z",
+     "start_time": "2025-03-05T18:51:44.757430Z"
+    }
+   },
+   "cell_type": "code",
+   "source": "!cat client/feature_store.yaml",
+   "id": "fac5f67ff391b5cf",
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "project: feast_rbac\r\n",
+      "provider: local\r\n",
+      "offline_store:\r\n",
+      "    host: localhost\r\n",
+      "    type: remote\r\n",
+      "    port: 8081\r\n",
+      "online_store:\r\n",
+      "    path: http://localhost:8082\r\n",
+      "    type: remote\r\n",
+      "registry:\r\n",
+      "    path: localhost:8083\r\n",
+      "    registry_type: remote\r\n",
+      "auth:\r\n",
+      "    type: kubernetes\r\n",
+      "entity_key_serialization_version: 3\r\n"
+     ]
+    }
+   ],
+   "execution_count": 7
+  },
+  {
+   "metadata": {},
+   "cell_type": "markdown",
+   "source": "**The Operator creates the client ConfigMap containing the feature_store.yaml. We can retrieve it and port froward to local**",
```

Comment:
what about `The Operator creates the ConfigMap containing the feature_store.yaml for a client application`

Reply:
can you explain this more,  this required `port-forward` to locally and feature_repo required to point feature-server to local accordingly.